MapboxAndroidDemo/src/main/AndroidManifest.xml

```diff
@@ -400,5 +394,23 @@
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value="com.mapbox.mapboxandroiddemo.MainActivity"/>
         </activity>
+        <activity
+            android:name=".account.LandingActivity"
+            android:screenOrientation="portrait">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN"/>
+                <category android:name="android.intent.category.LAUNCHER"/>
+            </intent-filter>
+            <action android:name="android.intent.action.VIEW"/>
```

Comment:
@langsmith I'm still seeing previous code without the `<intent-filter>`. Remember that there are 2 manifests `MapboxAndroidDemo/src/gpservices/AndroidManifest.xml` and `MapboxAndroidDemo/src/main/AndroidManifest.xml`. It seems only the first one was updated.